### PR TITLE
Update changelog for v7.0.0

### PR DESCRIPTION
**Pull request by harryi3t**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,16 +2,43 @@
...
+    - To execute a folder/request using a path
+    ```javascript
+    runner.run(collection, { entrypoint: {
+        execute: `${desiredId}`,
+        lookupStrategy: 'path',
+        path: ['grand_parent_folder_id', 'parent_folder_id']
+    }});
+    ```
+* [BREAKING] #428 Advanced auth flows are now enabled by default(`interactive` flag has been removed)
+* :tada: #424 Added support for collection level variables
```
Comment (kunagpal) [inline CHANGELOG.md]: This group can be compacted to one line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,16 +2,43 @@
...
-* [breaking] #453 #447 Added default timeout value to 3 min.
-    To get the earlier behaviour of never timeout by default, the option `timeout.global` needs to be passed as `0`.
-* [breaking] Updated `postman-sandbox` to `v3.0.0`, which now calls the assertion events for both new `pm.test` API and legacy `tests`.
-    All `assertion` events are now passed with an array of assertion. Also `tests` object has been removed from `result`.
-* #430 Added support for persistenting pm.variables which is available throughout the collection run
-* #427 Added support for filtering by request in entry  point
-* #424 Add support for collection level variables
-* #428 Enabled interactive auth by default
-* :bug: Fixed the case where setting entrypoint as an invalid value was throwing error
+* :bug: Invalid values in `entrypoint` now throw an error (when `abortOnError` is set to `true`)
```
Comment (kunagpal) [inline CHANGELOG.md]: This doesn't throw an error, it results in an error.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+* [BREAKING] #453 #447 Added default timeout value to 3 min.
+    - Timeout options take a positive value in milliseconds. e.g.
+        ```javascript
+        runner.run(collection, { timeout: {
+            global: 5 * 60 * 1000, // total timeout of 5 minutes
+            request: 500, // timeout for individual requests
+            script: 200 // timeout for individual scripts
+        } });
+        ```
+    - If you are increasing the default timeout for `script`/`request`, make sure `global` timeout is sufficiantly larger that that.
```
Comment (shamasis) [inline CHANGELOG.md]: than that

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+    - Use `0` for no timeout.
```
Comment (shamasis) [inline CHANGELOG.md]: `0` for no timeout (infinite timeout)

**Pull request merged**